opennms-provision/opennms-provision-persistence/src/main/resources/org/opennms/netmgt/provision/persist/default-foreign-source.xml

```diff
@@ -63,5 +63,6 @@
 		<detector name="MS-RDP" class="org.opennms.netmgt.provision.detector.simple.TcpDetector">
 			<parameter key="port" value="3389"/>
 		</detector>
+		<detector name="ActiveMQ" class="org.opennms.netmgt.provision.detector.jms.ActiveMQDetector"/>
```

Comment:
I'm not sure if we should add this to the default-foreign-source. Just from my personal point of view prefer a  "default-foreign-source" with no detectors at all.